packages/data-types/src/types/index.ts

```diff
@@ -0,0 +1,19 @@
+
+import * as ts from '@terascope/utils';
+import mappings from './versions/mapping';
+import { TypeConfig } from '../interfaces';
+
+export class TypesManager {
+    public version: string;
+
+    constructor(version: number) {
+        this.version = `v${version}`;
+        if (mappings[this.version] == null) throw new ts.TSError(`The type library does not have types for version ${version}`);
```

Comment:
This error message can be improved.

```suggestion
        if (mappings[this.version] == null) throw new ts.TSError(`Unknown DataType version ${version}`);
```